src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig

```diff
@@ -75,10 +77,14 @@
                 <span class="label">Sub-Request{{ profile.children|length > 1 ? 's' }}</span>
             </div>
 
-            {% set subrequests_time = 0 %}
-            {% for child in profile.children %}
-                {% set subrequests_time = subrequests_time + child.getcollector('time').events.__section__.duration %}
-            {% endfor %}
+            {% if has_time_events %}
+                {% set subrequests_time = 0 %}
+                {% for child in profile.children %}
+                    {% set subrequests_time = subrequests_time + child.getcollector('time').events.__section__.duration %}
+                {% endfor %}
+            {% else %}
+                {% set subrequests_time = 0 %}
```

Comment:
should it be n/a?

Reply:
The profiler panel shows 0 for all time values, only the toolbar shows n/a. I can change it to be n/a for all values if you like it better.

Reply:
"n/a" looks better to me